src/services/state.ts

```diff
@@ -149,7 +158,7 @@ function isCommentApplicable(comment: Comment, changes: Change[]): string[] {
     }
 
     // Check if any exclusion should filter out the match
-    for (const exclusion in exclusions) {
```

Comment:
Note: this caused exclusions to not work with the action. Tests could've helped!!